etc/schema/mysql-upgrade/v1.2.0.sql

```diff
@@ -17,9 +17,12 @@ ALTER TABLE x509_certificate
     CHANGE ca ca enum('n', 'y') NOT NULL,
     CHANGE trusted trusted enum('n', 'y') NOT NULL DEFAULT 'n';
 
-ALTER TABLE x509_certificate_chain CHANGE valid valid enum('n', 'y', 'yes', 'no') NOT NULL DEFAULT 'n';
+ALTER TABLE x509_certificate_chain
+    CHANGE valid valid enum('n', 'y', 'yes', 'no') NOT NULL DEFAULT 'n',
```

Comment:
Does this work with all MySQL/MariaDB versions we support? I would rather do this explicitly and call UPDATE instead of relying on this "magic".

Reply:
Nope, it won't work! I Just fucked up :)